### PR TITLE
CNCF annual report for review

### PR DESCRIPTION
**Pull request by Fei-Guo**
/kind documentation

This PR is created for review purpose.

### CODE REVIEW

File: docs/2021-OpenYurt-annual.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+It addresses specific challenges for cloud-edge orchestration in Kubernetes such as unreliable or disconnected
+cloud-edge networking, edge node autonomy, edge device management, region-aware deployment and so on.
+Currently, OpenYurt primarily provides the following capabilities:
+- A node daemon which proxies the network traffics from Kubelet and/or other daemons so that node components still
+  function even if the node is disconnected to the cloud.
+- A tunnel service to allow the cloud site control plane to access the node components running in the edge nodes.
+- A supplement to upstream controllers to deal with edge failures.
+- A workload for managing edge applications across multiple edge regions.
+- An integration with [EdgeX Foundry](https://github.com/edgexfoundry) to allow managing edge devices through Kubernetes.
+
```
Comment (rambohe-ch) [inline docs/2021-OpenYurt-annual.md:17]: how about add an introduction for `node-resource-manager`(local storage for edge) and `yurtcluster-operator`?

Comment (Fei-Guo) [reply docs/2021-OpenYurt-annual.md:17]: I have thought about it but feel they are implementation details that the ToC may not understand easily, especially the yurtcluster-operator.

Comment (rambohe-ch) [reply docs/2021-OpenYurt-annual.md:17]: @Fei-Guo OK, but how about add the following capabilities:
- ensure service access traffic(east-west traffic) closed in region(OpenYurt called NodePool)
- pods on the edge node can use InClusterConfig to access kube-apiserver directly without awareness cloud-edge network isolation.

File: docs/2021-OpenYurt-annual.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+We have established [the community membership roadmap](https://github.com/openyurtio/community/blob/main/community-membership.md), and quite a few active and qualified contributors
+have been promoted to maintainers. Compared to three maintainers (all from Alibaba) initially, we now have [eight maintainers](https://github.com/openyurtio/openyurt/blob/master/MAINTAINERS.md) from six organizations.
+- Chao Zheng (ByteDance)
+- Fei Guo (Alibaba)
+- Lifang Zhang (China Telecom)
+- Linbo He (Alibaba)
+- Shida Qiu (Alibaba)
+- Shaoqiang Chen (Intel)
+- Tao Chen (ZheJiang University)
+- Yixing Jia (VMware)
```
Comment (rambohe-ch) [inline docs/2021-OpenYurt-annual.md:49]: we can a new maintainer(@lwmqwer ) from VMware now

File: docs/2021-OpenYurt-annual.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  - We held bi-weekly community meetings constantly (total 32 as of Nov 2021). The meeting records can be found in [here](https://search.bilibili.com/video?keyword=openyurt). The average number of  meeting attendees is ~30.
+  - Number of contributors: 10+ -> **54**
+  - Github stars: 200+ -> **1000+**
+  - Github forks: 30+ -> **200+**
+  - Contributing organizations: 2 -> **22**
+
+## Maintainers
+
+We have established [the community membership roadmap](https://github.com/openyurtio/community/blob/main/community-membership.md), and quite a few active and qualified contributors
+have been promoted to maintainers. Compared to three maintainers (all from Alibaba) initially, we now have [eight maintainers](https://github.com/openyurtio/openyurt/blob/master/MAINTAINERS.md) from six organizations.
```
Comment (rambohe-ch) [inline docs/2021-OpenYurt-annual.md:41]: eight maintainers --> nine maintainers

**Pull request closed without merging**